File: 백준/Bronze/10818. 최소， 최대/최소， 최대.cc
# include <stdio.h>
# include <stdlib.h>

int main() {

	int n;
	int *arr;
	int max, min;
	scanf("%d", &n);


	arr = (int*)malloc(sizeof(int) * n);

	for (int i = 0; i < n; i++)
	{
		scanf("%d", &arr[i]);

	}

	min = arr[0];
	max = arr[0];

	for (int i = 0; i < n; i++)
	{
		if (max < arr[i])
		{
			max = arr[i];
		}

		if (min > arr[i])
		{
			min = arr[i];
		}
	}

	printf("%d %d", min, max);



	return 0;
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		
		int n = input.nextInt();
		int arr[] = new int[10000000];
		for (int i = 0; i < n; i++) {
			arr[i] = input.nextInt();
		}
		
		int min = arr[0];
		for (int j = 0; j < n; j++) {
			if (min > arr[j]) {
				min = arr[j];
			}
		}
		System.out.print(min+" ");
		
		int max = arr[0];
		for (int p = 0; p < n; p++) {
			if (max<arr[p]) {
				max = arr[p];
			}
		}
		System.out.println(max);
	}
}
